apps/vr-tests-react-components/src/stories/Spinner.stories.tsx

```diff
@@ -55,6 +55,10 @@ storiesOf('Spinner converged', module)
       includeDarkMode: true,
```

Comment:
Looks like the size tiny was modified? Also, would it be possible to add a wrapper to focus on the component itself?

Reply:
The tiny size appears to be from fixing the name of the `'Inverted with Size Tiny'` stories on line 175, and not an actual visual change.

But yes, it would be good to crop to the testWrapper.